.gitignore

```diff
@@ -59,5 +59,7 @@ ebin/*.beam
 rel/example_project
 .concrete/DEV_MODE
 .rebar
+rebar.lock
```

Comment:
According to the documentation, [`rebar.lock` should not be ignored](https://www.rebar3.org/docs/dependencies#section-lock-files). I suggest to commit this file.